aviary/mission/flops_based/phases/phase_utils.py

```diff
@@ -57,3 +57,15 @@ def add_subsystem_variables_to_phase(phase, phase_name, external_subsystems):
                 phase.add_path_constraint(constraint_name, **kwargs)
 
     return phase
+
+
+def get_initial(status, key, status_for_this_variable=False):
+    # Check if status is a dictionary.
+    # If so, return the value corresponding to the key or False if the key is not found.
```

Comment:
I think this should say "the default status" instead of "False", the usage in methods_for_level2 makes the default "True"